packages/core/src/domain/session/sessionManager.ts

```diff
@@ -53,6 +58,9 @@ export function startSessionManager<TrackingType extends string>(
     expireObservable.notify()
     sessionContextHistory.closeActive(relativeNow())
   })
+  sessionStore.sessionStateUpdateObservable.subscribe(({ previousState, newState }) =>
+    sessionStateUpdateObservable.notify({ previousState, newState })
+  )
```

Comment:
**💬 suggestion:** ‏no need to create a different `sessionStateUpdateObservable`, you can just return `sessionStore.sessionStateObservable`